source/features/i18n/translations/ja.json

```diff
@@ -128,7 +132,8 @@
     "notExist": "アドレスが見つかりません",
     "notFound": "一致する検索結果はありません",
     "placeholder": "エポック、ブロック、アドレス、トランザクションを検索する",
-    "suggestion_block": "ブロックを検索",
+    "suggestion_block_by_number": "<<>>",
```

Comment:
@eddyjlhaigh Please provide translations for

- _Search for a block by number_
- _Search for a block by slot number_